### DISCUSSION TITLE
Reannouncement on Award Session for HW4

### DISCUSSION THREAD

**Discussion by spearo2** [prosyslab-classroom/cs424-program-reasoning#444] [Announcements]
안녕하세요. 조교 이창공 입니다.

수상자분들이 발표를 더 준비할 수 있도록
HW4의 시상식 일정을 다음주 수요일(11/27)로 변경하겠습니다. 

수상자 분들은 과제 마감기간 이후에 메일로 통보 드리도록 하겠습니다.

감사합니다.

---

Hello,

To give the awardees more time to prepare their presentations, the HW4 award session has been rescheduled to next Wednesday (November 27).

The awardees will be notified via email after the assignment deadline.

Thank you.

**Comment by KihongHeo** [comment]
Oh, `etyu` eventually outperforms the previous champions. Big congratulations.

**Reply by etyu39** [reply]
Thank you professor!
